sklearnex/preview/covariance/covariance.py

```diff
@@ -100,4 +106,27 @@ def fit(self, X, y=None):
 
         return self
 
+    # This needs to be included to guarantee that we use the sklearnex pairwise_distances
+    @wrap_output_data
+    def mahalanobis(self, X, queue=None):
```

Comment:
Could you please clarify, why queue is here?

Reply:
I don't like this API inconsistency for `fit` and `mahalanobis`

Reply:
I have removed the queue. pairwise_distances in daal4py has input checks within it following daal4py style.